apps/betterangels-backend/notes/tests/test_queries.py

```diff
@@ -192,57 +192,121 @@ def test_notes_paginated_query(self) -> None:
-            # CM 2 has one note "deets" for client "truman"
-            ("org_1_case_manager_2", None, "deets rum", None, 1, "note_3", None),
-            ("org_1_case_manager_2", None, "deets rum", True, 0, None, None),  # CM 2 has no submitted notes
-            ("org_1_case_manager_1", "client_user_2", None, None, 0, None, None),  # CM 1 has no notes for client 2
-            # CM 2 has one unsubmitted note for client 1
-            ("org_1_case_manager_2", "client_user_1", None, False, 1, "note_2", None),
-            (None, None, None, True, 0, None, None),  # There are no submitted notes
-            (None, None, None, None, 3, False, None),  # There are three unsubmitted notes
+            # created by, client_label, and/or is_submitted
+            ("org_1_case_manager_1", None, None, 1, ["note"]),  # CM 1 created one note
+            ("org_1_case_manager_2", None, None, 2, ["note_2", "note_3"]),  # CM 2 created 2 notes
+            ("org_1_case_manager_2", None, False, 1, ["note_2"]),  # CM 2 has one unsubmitted note
+            ("org_1_case_manager_2", None, True, 1, ["note_3"]),  # CM 2 has one submitted note
+            ("org_1_case_manager_1", "client_user_2", None, 0, []),  # CM 1 has no notes for client 2
+            ("org_1_case_manager_1", "client_user_1", False, 1, ["note"]),  # CM 1 has one unsubmitted note for client 1
+            (None, None, False, 2, ["note", "note_2"]),  # There are two unsubmitted notes
         ],
     )
     def test_notes_query_filter(
         self,
         case_manager_label: Optional[str],
         client_label: Optional[str],
-        search_terms: Optional[str],
         is_submitted: Optional[bool],
         expected_results_count: int,
-        returned_note_label_1: Optional[str],
-        returned_note_label_2: Optional[str],
+        returned_note_labels: list[str],
+    ) -> None:
+        self.graphql_client.force_login(self.org_1_case_manager_2)
+        # self.note is created in the setup block by self.org_1_case_manager_1 for self.client_user_1
+        self.note_2 = self._create_note_fixture({"purpose": "Client 1's Note", "client": self.client_user_1.pk})[
+            "data"
+        ]["createNote"]
+        self.note_3 = self._create_note_fixture({"purpose": "Client 2's Note", "client": self.client_user_2.pk})[
+            "data"
+        ]["createNote"]
+        self._update_note_fixture(
+            {
+                "id": self.note_3["id"],
+                "isSubmitted": True,
+            }
+        )
+
+        query = """
+            query Notes($filters: NoteFilter) {
+                notes: notesPaginated(filters: $filters) {
+                    totalCount
+                    results{
+                        id
+                    }
+                }
+            }
+        """
+
+        filters: dict[str, Any] = {}
+
+        if case_manager_label:
+            filters["createdBy"] = getattr(self, case_manager_label).pk
```

Comment:
i saw recently that there's already a user_map keyed by label:
```suggestion
            filters["createdBy"] = self.user_map[case_manager_label].pk
```